src/lib/konnectors.js

```diff
@@ -55,6 +60,10 @@ export function isKonnectorLoginError(error) {
   return error && error.type && error.type === ERROR_TYPES.LOGIN_FAILED
 }
 
+export function isKonnectorTwoFAError(error) {
```

Comment:
Nit: When I use helpers file I like to do

`import { * as konnectors} from './helpers/konnectors`

so that I do not have to repeat `Konnector` in all my methods.

`konnectors.isTwoFAError`

Reply:
I would suggest to keep it like this for now since it's already using this naming everywhere else and I would like to avoid regressions in this PR

Reply:
I made an issue https://github.com/cozy/cozy-home/issues/1224 to keep tracking it